Source/Scene/BingMapsImageryProvider.js

```diff
@@ -533,7 +553,21 @@ define([
         }
         //>>includeEnd('debug');
 
-        return ImageryProvider.loadImage(this, buildImageResource(this, x, y, level, request));
+        var promise = ImageryProvider.loadImage(this, buildImageResource(this, x, y, level, request));
+
+        if(defined(promise)) {
+            return promise.then(() => promise, (error) => {
+                // If there isn't any image, we get an InvalidStateError.
+                // The version of the Bing maps API that we hit if we're using `BingMapsStyle.AERIAL_WITH_LABELS_ON_DEMAND` does this
+                // instead of returning a "This image is missing" image.
+                // Since this isn't really an error, we want to supress the one that just got thrown.
+                if (error.name === 'InvalidStateError' && error.message === 'The source image could not be decoded.') {
```

Comment:
This test isn't great, because there are other reasons that image decoding could fail. I think the best solution is to change `Resource.prototype.fetchImage` to attach the `generatedBlob` to the `error` object in the `otherwise` block. Then you can check for it on the `error` and check if its size is zero.